include/ibamr/FESurfaceDistanceEvaluator.h

```diff
@@ -62,11 +62,11 @@ class FESurfaceDistanceEvaluator
     /*!
      * The only constructor of this class.
      */
-    FESurfaceDistanceEvaluator(const std::string& object_name,
+    FESurfaceDistanceEvaluator(std::string object_name,
                                SAMRAI::tbox::Pointer<SAMRAI::hier::PatchHierarchy<NDIM> > patch_hierarchy,
                                SAMRAI::tbox::Pointer<IBAMR::IBFEMethod> ibfe_method,
-                               const libMesh::Mesh& mesh,
-                               const libMesh::BoundaryMesh& bdry_mesh,
+                               libMesh::Mesh mesh,
+                               libMesh::BoundaryMesh bdry_mesh,
```

Comment:
@abarret  and @drwells: This change from const & to pure object is causing problems down the lane when I do element iterator on mesh and bdry_mesh. Can I change it back to const Mesh& and const BoundaryMesh& ?

Reply:
Yes. This may have been an over zealous use of the new `move` syntax.